setup.py

```diff
@@ -36,6 +37,7 @@ def read_file(filename, encoding='utf8'):
     'pyasn1',  # urllib3 InsecurePlatformWarning (#304)
     'pycrypto',
     'PyOpenSSL',
+    'pyparsing>=1.5.5',
```

Comment:
Strict pinning (`==`) often leads to problems such as https://github.com/Demonware/jose/pull/1 or https://github.com/fatiherikli/nginxparser/pull/8, etc. setup.py version (non-strict) pins should be rather based on the availability of advertised API functions (hence `>=` is a "safe choice", i.e. provided dependent libraries are maintained in backwards-compatible fashion, which is also sane assumption most of the time). requirements.txt with strict pins could be treated as so called Known Good Set and we might use it later, when things settle down. This kind of setup can be found e.g. in Plone.